shotover-proxy/example-configs/cassandra-cluster-multi-rack/docker-compose.yml

```diff
@@ -11,7 +11,7 @@ networks:
 
 services:
   cassandra-one:
-    image: bitnami/cassandra:4.0.6
+    image: bitnami/cassandra:4.0
```

Comment:
is this change intentional?

Reply:
This set off a rampage of adding anchors wherever I could 😂

Reply:
:tada: